Update Kvatch Rebuilt

charabaruk opened the pull request:
Add CRC for cleaned 3.0RC3. Add optional "no more burned ground" additional plugin. Add All Natural weather patch for KR.

charabaruk committed:
Update Kvatch Rebuilt

Add CRC for cleaned 3.0RC3. Add optional "no more burned ground"
additional plugin. Add All Natural weather patch for KR.

sibir-ine commented on masterlist.yaml:
```diff
@@ -12959,6 +12961,9 @@ plugins:
           - 'Unique Landscapes - Merged ESP'
           - '[Unique Landscapes Compatibility Patches](https://www.nexusmods.com/oblivion/mods/13834/)'
         condition: 'version("Unique Landscapes.esp", "2.0.0", >=) and active("Unique Landscapes.esp") and not file("ULE KvatchRebuilt-CheydinhalFalls patch.esp")'
+      - <<: *patchIncluded
+        subs: [ 'Weather - All Natural' ]
+        condition: 'active("All Natural Base.esm") and not active("Kvatch Rebuilt Weather Patch.esp")'
```
```suggestion
```

The text of `patchIncluded` is "You seem to be using **%1%**, but you have not enabled a compatibility patch for this mod. A compatibility patch is included with this plugin''s installer." This patch isn't included in _Kvatch Rebuilt_'s installer, but rather _Weather - All Natural_'s installer, so the message should be applied to it instead. You can do so in this PR if you want.

charabaruk replied:
Ah, I wasn't sure which it should have gone into.

sibir-ine commented on masterlist.yaml:
```diff
@@ -12970,6 +12975,16 @@ plugins:
         crc: 0x50EEF62E
         util: '[TES4Edit v4.0.3](https://www.nexusmods.com/oblivion/mods/11536)'
         itm: 138
+    clean:
+      - crc: 0x489D7BE6
+        util: 'TES4Edit v4.0.4c'
```
```suggestion
```

We don't include cleaning data for dirty mods after they're cleaned since it's redundant (users will know if they cleaned the mod themselves) & volatile (different xEdit versions & QAC settings can produce different checksums).

sibir-ine commented on masterlist.yaml:
```diff
@@ -12970,6 +12975,16 @@ plugins:
+    clean:
+      - crc: 0x489D7BE6
+        util: 'TES4Edit v4.0.4c'
+  - name: 'Kvatch Rebuilt - No More Burned Ground.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/15412/'
+        name: 'Kvatch Rebuilt'
+    req: [ 'Kvatch Rebuilt.esp' ]
+    msg:
+      - <<: *optional
```
```suggestion
```

This plugin already has `Kvatch Rebuilt.esp` as a master, so a requirement is redundant. `optional` is also only for optional plugins that are packaged with the main plugin & that mod managers will therefore install by default, informing users that they may not actually need or want it. That leaves the location which isn't useful on its own, so the plugin can be removed (unless you want to add other metadata for it).

charabaruk replied:
Replaced with the dirty data for the plugin. Considering that it's an additional download from Kvatch Rebuilt rather than its own mod, I think it's probably helpful to have the link in there for the no more burned ground esp.

charabaruk committed:
Move Weather - All Natural / Kvatch Rebuilt patch message

Since the patch is from Weather - All Natural, not in Kvatch Rebuilt,
the message should be in the former.

charabaruk committed:
Update Kvatch Rebuilt data

Remove self-clean data from running Kvatch Rebuilt through QAC, and add
dirty data for the 'no more burned ground' additional plugin in place of
unnecessary req and message data.

sibir-ine commented on masterlist.yaml:
```diff
@@ -4236,6 +4236,9 @@ plugins:
         condition: 'active("DLCShiveringIsles.esp") and not active("All Natural - SI.esp")'
+      - <<: *patchIncluded
+        subs: [ 'Kvatch Rebuilt' ]
+        condition: 'active("Kvatch Rebuilt.esp") and not active("Kvatch Rebuilt Weather Patch.esp"'
```
```suggestion
        condition: 'active("Kvatch Rebuilt.esp") and not active("Kvatch Rebuilt Weather Patch.esp")'
```

The condition is missing a closing parenthesis.

charabaruk replied:
Oops, dumb mistake. 🤦‍♂️

charabaruk committed:
fix syntax error in condition
